app/assets/stylesheets/show_styles.css

```diff
@@ -32,3 +32,8 @@
   height: 20px;
   width: 15px;
 }
+
+.review_radio
```

Comment:
I didn't notice this before, but it would be a good idea to clean up the way these styles are written--it's a lot more common to see this kind of format:

```
.review_radio {
  display: inline;
}
```